functions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+function bottlesHtml(array $allBottles): string {
+
+    //check to see whether this is an array of arrays and gracefully exit if not
+    if (count($allBottles) == count($allBottles, COUNT_RECURSIVE))
+    {
+        return "bottlesHTML function has not been passed an array within an array";
+    }
+    $unfriendlyNames = ["purchaselocation", "type", "purchasedate"];
+    $friendlyNames = ["Purchase location", "Type", "Date purchased"];
+    $bottleHtml = "";
```

Comment:
i would rename this to `bottlesHtml` just because it has the html for all the bottles and not just one